ui/src/ScholarReader.tsx

```diff
@@ -412,28 +413,31 @@ class ScholarReader extends React.PureComponent<ScholarReaderProps, State> {
   }
 
   scrollSymbolIntoView() {
-    setTimeout(() => {
-      const { selectedSymbol, pdfViewer, pages } = this.state;
-      if (pdfViewer && selectedSymbol) {
-        const symBounds = selectedSymbol.bounding_boxes[0];
-        const { left } = selectors.divDimensionStyles(
-          pages[symBounds.page + 1].view, symBounds
-        );
-        
-        // Left = left position on the pdf page, scroll left = left offset of the pdf
-        // 200 = extra padding from the pdf side panel. 
-        const relativeSymbolLeftPosition = left - pdfViewer.container.scrollLeft + 200;
-        // window.innerWidth = possible visible area of the viewport, 
-        // 470 = width now obscured by the open drawer. 
-        const viewableViewportWidth = window.innerWidth - 470;
-        console.log(`Left Position: ${relativeSymbolLeftPosition} > ${viewableViewportWidth} | moving: ${relativeSymbolLeftPosition - viewableViewportWidth}`);
-        if (relativeSymbolLeftPosition > viewableViewportWidth) {
-          // Here we are moving the pdf to the left by the amount the symbol was obscured by
-          // + some padding to make it a little bit away from the drawer. 
-          pdfViewer.container.scrollLeft += Math.max((relativeSymbolLeftPosition - viewableViewportWidth) + 50, 0);
-        }
-      } 
-    }, 0);
+    const { selectedSymbol, pdfViewer, pages } = this.state;
+    if (pdfViewer && selectedSymbol) {
+      const symBounds = selectedSymbol.bounding_boxes[0];
+      const { left } = selectors.divDimensionStyles(
+        pages[symBounds.page + 1].view, symBounds
+      );
+      /*
+      * Each component of the calculation: 
+      * Left = left position on the pdf page of the selected symbol
+      * scrollLeft = how much the pdf has been scrolled left already
+      * 200 = the extra padding from the pdf's side panel
+      * ----------------
+      * innerWidth = possible visible area of the viewport
+      * 470 = width of the drawer that is now obscuring the view
+      */
+      /*
+      * TODO(@zkirby) need to account for if the side panel is not open.
+      */
+      const relativeSymbolLeftPosition = left - pdfViewer.container.scrollLeft + 200;
```

Comment:
Can we move these magic numbers into named constants?

Reply:
And can we make the `470` the same variable as wherever else it's set in the code?

Reply:
Should these be moved to a constants.js file?

Reply:
So far, I've just been assigning them meaningful names, putting them in a file they feel they most belong in, and moving them to the headers of the file. Would that work?

Reply:
Also the 470 comes from some less variables, is there an easy way to import these into JS? Everything online requires installing npm packages

Reply:
I'm not sure! If it requires an extra npm package, then maybe we should skip it, and just note in the comments that the two values are the same and should be changed together?

Reply:
I'll point out where it comes from in the comments and if we run into this issue in the future maybe we can consider adding the package?

Reply:
Sounds good!